tools/lib/gzipTileset.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+var Cesium = require('cesium');
+var fsExtra = require('fs-extra');
+var path = require('path');
+var Promise = require('bluebird');
+var zlib = require('zlib');
+
+var fsExtraCopy = Promise.promisify(fsExtra.copy);
+var fsExtraEnsureDir = Promise.promisify(fsExtra.ensureDir);
+var fsExtraReadFile = Promise.promisify(fsExtra.readFile);
+
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = gzipTileset;
+
+/**
+ * gzips or gunzips the input tileset.
+ *
+ * @param {String} inputDirectory Path to the input directory.
+ * @param {Object} [outputDirectory] Path to the output directory.
+ * @param {Object} [options] Object with the following properties:
+ * @param {Boolean} [options.gzip=true] Whether to gzip or gunzip the tileset.
```

Comment:
Many unix command line tools do ask for permission before overwriting. Node libraries on the other hand like `fs` always overwrite. If I were to chose I prefer the overwrite always way.